### PR TITLE
Add new metrics to show capacity and items across objects

### PR DESCRIPTION
**Pull request by YueTang-Vanessa**
Add two metrics `bloom_num_items_across_objects` and `bloom_capacity_across_objects` into the info section.

 * bloom_num_items_across_objects : indicate current total number of items across bloom filter objects
 * bloom_capacity_across_objects: indicate current total capacity across bloom filter objects

### CODE REVIEW

**Review by zackcam — COMMENTED** [review]
I think we also need to update [this](https://github.com/valkey-io/valkey-bloom/blob/unstable/src/bloom/data_type.rs#L114). test_save_and_restore_metrics should be failing but isnt because of [self.client](https://github.com/valkey-io/valkey-bloom/blob/unstable/tests/test_bloom_metrics.py#L141)  being used and not new_client

**Comment by YueTang-Vanessa** [comment]
> I think we also need to update [this](https://github.com/valkey-io/valkey-bloom/blob/unstable/src/bloom/data_type.rs#L114). test_save_and_restore_metrics should be failing but isnt because of [self.client](https://github.com/valkey-io/valkey-bloom/blob/unstable/tests/test_bloom_metrics.py#L141) being used and not new_client

The `BloomFilter::from_existing()` will change the metric

File: src/metrics.rs [hunk 1]
```diff
@@ -27,6 +29,18 @@ pub fn bloom_info_handler(ctx: &InfoContext) -> ValkeyResult<()> {
...
+            BLOOM_NUM_ITEMS_ACROSS_OBJECTS
+                .load(Ordering::Relaxed)
+                .to_string(),
+        )?
+        .field(
+            "bloom_capacity_across_objects",
+            BLOOM_CAPACITY_ACROSS_OBJECTS
+                .load(Ordering::Relaxed)
+                .to_string(),
+        )?
```
Comment (KarthikSubbarao) [inline src/metrics.rs:43]: These metrics will also be incremented during RDB Load. Can we test this out in (1) RDB Load restore test (2) COPY command test?

Comment (YueTang-Vanessa) [reply src/metrics.rs:43]: RDB load will use from_existing function where the metrics get increased. The tests been covered in the test_bloom_metrics.py test_save_and_restore_metrics and test_copy_metrics

**Review by KarthikSubbarao — APPROVED** [review]

**Pull request merged**